Depth map completed

mansi75 opened the pull request:
KIndly merge the pull request

geekquad reviewed (changes requested):
Accepting only .ipython notebooks.

geekquad commented on Riddler-Calvard/riddler_calvard.ipynb:
```diff
@@ -0,0 +1,118 @@
+        "outputId": "ae182f00-cc5c-418f-e94b-9ce87ba45993"
+      },
+      "source": [
+        "!pip install mahotas\n"
```
```mahotas``` not allowed. 
Please red the Contributing Guidelines.

mansi75 replied:
@geekquad Riddler-calvard without mahotas?

geekquad replied:
Our focus is to have various feature implementations of OpenCV in Python.

mansi75 commented:
@geekquad Changes made in depth notebook

geekquad reviewed (changes requested):
Please add the sample input image in the notebook as well.

mansi75 commented:
@geekquad Kindly check again, I have done it

geekquad reviewed (commented):
1. Depth Map is okay.
2. I cannot see any Image Processing technique being used in Geometric shapes.
3. If mahotas is not being used, please omit that from the codebase.
4. I cannot merge specific files from the PR as such, make different branches for each of the issues.

mansi75 commented:
1. In Geometric shapes, I have used OpenCV methods cv2.line, cv2.rectangle, cv2.circle etc to draw various shapes on images. Kindly check I have updated it too. 
2. I have removed riddler calvard from the repo since ridler calvard can not be used without mahatos.

geekquad commented:
Hey @mansi75, reverted you back. 
Kindly check the discord server.

mansi75 commented:
@geekquad 
Kindly check again

geekquad reviewed (approved):
Heu @mansi75, good work!
